.ci/run_cc_tests.sh

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2017 Intel Corporation
```

Comment:
This is script that will be common to all testing, should be in the `tests` repo.